doc/api/util.md

```diff
@@ -666,7 +666,7 @@ Deprecated predecessor of `console.log`.
 
 Deprecated predecessor of `console.log`.
 
-### util._extend(obj)
+### util._extend(targetObj, sourceObj)
```

Comment:
why not `util._extend(target, source)` without `obj` suffix ? @yorkie `origin/add` combination is confusing.